### DISCUSSION TITLE
java 8

### DISCUSSION THREAD

**Discussion by AvishavTal** [devopvoid/webrtc-java#93] [Q&A] [answered]
is there a way to use this library in a java 8 project?

**Comment by devopvoid** [comment]
It is possible, for that I have to create a multi-version release.

Maybe it's easier for you to update to Java 11? :-)

**Reply by AvishavTal** [reply]
I need it for embedded machines that don't support Java 11, so unfortunately it doesn't solve my problem.
Is multi-version release in your future plans? 
Or is there mybe another solution?

**Reply by AmramVaknin** [reply]
Will be very helpful 🙏

**Reply by devopvoid** [reply]
I'll try to create a multi-release with the next version

**Reply by AvishavTal** [reply]
That will be great! Thanks

**Reply by devopvoid** [reply]
The latest 0.7.0-SNAPSHOT should run within a java 8 project.

**Comment by AmramVaknin** [comment]
Amazing!

Thanks for this powerful project

**Comment by AmramVaknin** [comment]
Thank you very much, really appreciate  it
Amram

‫בתאריך יום ד׳, 9 בנוב׳ 2022 ב-9:48 מאת ‪Alex Andres‬‏ <‪
***@***.***‬‏>:‬

**Comment by devopvoid** [comment, marked as answer]
Pushed version 0.7.0 to Maven Central. If this works for you guys, please mark this topic as 'answered'.

**Reply by AvishavTal** [reply]
It works! thank you very much!
